plugins/modules/accesspoint_workflow_manager.py

```diff
@@ -103,14 +103,20 @@
         required: False
       ap_mode:
         description: |
-          Mode of operation for the Access Point (AP). Possible values include "local/flexconnect",
-          "monitor", "sniffer", or "Bridge/Flex+Bridge".(eg. "Local")
+          Mode of operation for the Access Point (AP). Possible values include "Local",
```

Comment:
`    Defines the mode of operation for the Access Point (AP). Possible values include "Local", "Monitor", "Sniffer", or "Bridge". For example, "Local".
`